starwars/src/components/CharButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Char  from './Char.js'
+
+const CharButton = (props)=> {
+
+console.log(props.nameprops);
+  return(
+
+    <div>
+      {props.nameprops.map(name => <Char nameprops={name} />)}
```

Comment:
Going off of what I suggested in your App.js file, how could you change the naming conventions here to describe the data better?